spec/models/form526_submission_spec.rb

```diff
@@ -23,6 +23,40 @@
     File.read('spec/support/disability_compensation_form/submissions/only_526.json')
   end
 
+  describe 'submit_endpoint enum' do
+    context 'when submit_endpoint is evss' do
+      let(:submit_endpoint) { 'evss' }
+
+      it 'is valid' do
+        expect(subject).to be_valid
+      end
+    end
+
+    context 'when submit_endpoint is claims_api' do
+      let(:submit_endpoint) { 'claims_api' }
+
+      it 'is valid' do
+        expect(subject).to be_valid
+      end
+    end
+
+    context 'when submit_endpoint is benefits_intake_api' do
+      let(:submit_endpoint) { 'benefits_intake_api' }
+
+      it 'is valid' do
+        expect(subject).to be_valid
+      end
+    end
+
+    context 'when submit_endpoint is not evss, claims_api or benefits_intake_api' do
+      it 'is invalid' do
+        expect do
+          subject.submit_endpoint = 'other_value'
```

Comment:
So this is the only place I see the subject attribute actually getting set in these tests, I don't think the let blocks above for `let(:submit_endpoint)` are doing anything?

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/16693/files#r1602031783